betelgeuse.py

```diff
@@ -574,6 +621,40 @@ def cli(context, jobs):
     testimony.SETTINGS['minimum_tokens'] = ['id']
 
 
+@cli.command('create-plan')
+@click.option(
+    '--name',
+    default='test-plan-{0}'.format(time.time()),
+    help='Name for new Test Plan.',
+)
+@click.option(
+    '--type',
+    default='release',
+    help='Test Plans can be Releases or Iterations.',
+    type=click.Choice([
+        'release',
+        'iteration',
+    ])
+)
+@click.option(
+    '--parent-name',
+    help='Name of parent Test Plan to link to.',
+)
+@click.option(
+    '--collect-only',
+    help=('Not perform any operation on Polarion, just prints '
```

Comment:
Since this is already inside a parenthesis context the ones around the string can be dropped and the string can still be break in more lines.